src/index.ts

```diff
@@ -29,28 +29,35 @@ const token = process.env.TOKEN;
 let timer = 0;
 
 // condition validation
+const checkMentions = (message: Message, checkId: string): boolean => {
+  return message.mentions.has(checkId);
+};
+
```

Comment:
```suggestion
const checkMentions = (message: Message, checkId: string): boolean => {
  return message.content.toString().includes(checkId);
};
```
Pulling this down locally, the `has` method isn't firing correctly on the `id` string. It looks like the docs say it requires a UserResolvable object. We could optionally fetch the users based on ID if needed. However, instead of that, if we cast the message content into a string then mentions are converted to `<@!id>` format and the `includes` method will catch them correctly.

Thoughts?

Reply:
@nhcarrigan Looking at the UserResolvable docs,
![image](https://user-images.githubusercontent.com/49726028/102264429-3998b080-3edb-11eb-8460-9638cd303005.png)
We can pass in a user object or a `Snowflake`, which is an id. Not sure why it's not working though.

Has the casting method worked for you in your testing?

Reply:
The casting method did work for me, yes.

The problem with UserResolvable is that a Snowflake is different from a string - we'd have to parse the `id` string into a Snowflake object.

Reply:
Oh, ok. I'll commit your suggestion then.